corelib-definitions/src/main/java/eu/europeana/corelib/definitions/edm/model/metainfo/WebResourceMetaInfo.java

```diff
@@ -0,0 +1,9 @@
+package eu.europeana.corelib.definitions.edm.model.metainfo;
+
+public interface WebResourceMetaInfo {
+    public String getId();
```

Comment:
Interfaces do not require the access level as they are always public. Remove public from all methods in this interface.